srcpkgs/octave/template

```diff
@@ -1,19 +1,33 @@
 # Template file for 'octave'
 pkgname=octave
-version=5.2.0
-revision=2
+version=6.2.0
+revision=1
 build_style=gnu-configure
-configure_args=" --with-blas=openblas --with-lapack=openblas"
-hostmakedepends="perl gcc-fortran pkg-config gnuplot"
+hostmakedepends="perl gcc-fortran pkg-config gnuplot texinfo texlive-dvi tar"
 makedepends="pcre-devel openblas-devel readline-devel libSM-devel libltdl-devel
  lcms2-devel glpk-devel"
-depends="$(vopt_if gui qt5-plugin-sqlite) texinfo"
+depends="$(vopt_if gui qt5-plugin-sqlite) texinfo texlive-dvi tar"
+checkdepends="texlive-dvi tar zip unzip"
 short_desc="High-level language, primarily intended for numerical computations"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="GPL-3.0-or-later"
 homepage="https://gnu.org/software/octave/"
+changelog="https://www.gnu.org/software/octave/NEWS-6.1.html"
 distfiles="${GNU_SITE}/octave/octave-${version}.tar.gz"
-checksum=2fea62b3c78d6f38e9451da8a4d26023840725977dffee5250d3d180f56595e1
+checksum=457d1fda8634a839e2fd7cfc55b98bd56f36b6ae73d31bb9df43dde3012caa7c
+
+# Use OpenBLAS on platforms where it is available and fallback to regular BLAS
+# on all others.
+case "$XBPS_TARGET_MACHINE" in
+	# List of supported architectures copied from openblas.
+	armv[67]*|aarch64*|i686*|x86_64*|ppc64*)
```

Comment:
why add this list now?after taking a look at openblas/template, openblas is available for ppc and armv5 as well and they aren't listed (meaning: all archs Void provides binary packages for have openblas available).
you might as well be better off just marking `mips*)` to use regular blas to match this, as done in openblas/template.

may i ask which arch you had in mind in specific which caused any trouble here?

Reply:
openblas has:

```
case "${XBPS_TARGET_MACHINE}" in
	armv5*) make_build_args+=" TARGET=ARMV5" ;;
	armv6*) make_build_args+=" TARGET=ARMV6" ;;
	armv7*) make_build_args+=" TARGET=ARMV7" ;;
	aarch64*) make_build_args+=" TARGET=ARMV8 DYNAMIC_ARCH=1" ;;
	i686*) make_build_args+=" BINARY=32 TARGET=GENERIC DYNAMIC_ARCH=1" ;;
	x86_64*) make_build_args+=" BINARY=64 TARGET=GENERIC DYNAMIC_ARCH=1" ;;
	ppc64le*) make_build_args+=" TARGET=POWER8 DYNAMIC_ARCH=1" ;;
	ppc64*) make_build_args+=" TARGET=PPC970MP" ;; # dynamic arch broken for <power6
	ppc*) make_build_args+=" TARGET=PPCG4" ;;
	mips*) broken="Not supported" ;;
	*) broken="Add your CPU" ;;
esac
```

which would mean that eventual `riscv` target would be broken by default, for example, and would block octave from being built instead of building it with `lapack`. I think listing where it's known to work is a bit more correct.